hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+userInput.addEventListener("keydown", handleKeyDown);
```

Comment:
The search seems to be a character behind. For example, if I type "times", "time" will be highlighted until I press spacebar then "times" will be fully highlighted. I think the issue could be in the code updating only during a 'keydown' event and not on input. I changed "keydown" to "input" and the code worked as intended, although the function should also be switched to handleInput for clarity.